pyramid_oereb/contrib/data_sources/standard/sources/plr.py

```diff
@@ -489,11 +492,12 @@ def read(self, params, real_estate, bbox):  # pylint: disable=W:0221
         """
         # Check if the plr is marked as available
         if self._is_available(real_estate):
-            session = self._adapter_.get_session(self._key_)
+            session = self.get_session()
+            empty_records = [EmptyPlrRecord(Config.get_theme_by_code_sub_code(self._plr_info['code']))]
```

Comment:
I find this refactoring slightly risky. There is no problem in the current use situation, but it can lead to a similar bahaviour as mutable default values for functions (see [pylint W0102](http://pylint-messages.wikidot.com/messages:w0102) or https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil/)
Here the mutable array is only assigned to one variable (either in `if` or in `else`), but there is some risk that code refactoring might lead to the variable `empty_records` being used more than once, with undesirable side effects.

If it's important for you to refactor the code, you might consider using a local factory function like:
```
def generate_empty_records():
    return [EmptyPlrRecord(Config.get_theme_by_code_sub_code(self._plr_info['code']))]
```
then you can use it by calling it `generate_empty_records()`

Reply:
I reverted this-you are right.